kimenyikevin/FreeMentor
# #168236083 Add logo on UI to be available for all pages

#### What does this PR do?
- Show logo on all pages
#### Description of Task to be completed?
- have all page display logo
#### How should this be manually tested?
- you can use gh-pages to verify UI  https://kimenyikevin.github.io/FreeMentor/UI
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168236083
#### Screenshots (if appropriate)
![ui-logo](https://user-images.githubusercontent.com/52419102/64075265-d6671880-ccad-11e9-90fd-83f4aee8673c.PNG)

#### Questions:
